service/src/test/java/com/ge/predix/acs/service/policy/evaluation/SubjectAttributeResolverTest.java

```diff
@@ -115,36 +106,11 @@ public void testGetSubjectAttributesSubjectFoundWithAttributesAndNoSupplementalA
 
     @Test
     public void testGetSubjectAttributesSupplementalAttributesOnly() {
-        BaseSubject testSubject = new BaseSubject();
-        testSubject.setSubjectIdentifier("/test/subject");
-
         Set<Attribute> supplementalSubjectAttributes = new HashSet<>();
         supplementalSubjectAttributes.add(new Attribute("https://acs.attributes.int", "site", "sanramon"));
```

Comment:
@FrankGasparovic , this test is asserting that if subject doesn't have any attributes but supplemental attributes are passed in, then supplemental attributes are returned by the attribute resolver.